python/votingapp/README.md

```diff
@@ -0,0 +1,106 @@
+# Voting App
+
+The votingapp demostrates how you can create a Fission function to connect to a relational database like Postgres and perform SQL operations. This is a basic app to help you understand how you can leverage Fission functions for working with databases.
+
+# Pre Requisites
+
+## Installing PostgreSQL
+Before you start working on this demo, you need to ensure that you have Fission installed and running on your system. You can refer to our [Fission Installation](https://fission.io/docs/installation/) guide for more. Further, this example uses PostgreSQL hence you must also have a PostgreSQL instance in place. You can either run it on your system locally, in a docker container or your Kubernetes cluster. Below are few guides you can refer to for installing PostgreSQL.
+
+- Installing Postgres SQL on your local system - <a href="https://www.postgresql.org/download/" target="_blank">Postgresql.org</a>
+- Postgres Docker container - <a href="https://hub.docker.com/_/postgres" target="_blank">Docker Hub</a>
+- Postgres HELM charts - <a href="https://bitnami.com/stack/postgresql/helm" target="_blank">Bitnami</a>
+
+<br>
+
+> *Note: If you have installed it using Helm Charts, please ensure to note down the service url and port number. This will be used to connect to the database from the function. You can get it by running  `helm status postgresql` in your terminal.*
+
+<br>
+
+### Database and Table Setup
+For this example, we have created a database named `votedb` and a table named `votebank`. You can use the following query to create the table
+
+```  
+ CREATE TABLE votebank (
+        id serial PRIMARY KEY,
+	    voter_id VARCHAR ( 50 )  NOT NULL,
+	    vote VARCHAR ( 50 ) NOT NULL
+    );
+```
+
+## Environment Setup
+<br>
+
+### Python Environment setup
+Using PostgreSQL with Fission requires certain libraries and Python modules to be present in the Python environment. For that you need to create a custom Python image. To do this you can refer to [Fisson Python Environment](https://github.com/fission/environments/tree/master/python) and follow the steps to create a custom image.
```

Comment:
So python modules needed can be satisfied by the builder. But if python requirements, requires C extensions or non-standard python dependencies, those needs to be installed by customizing environment image.

Eg. postgresql-dev needs to be installed by customizing the env image. 
But dependencies like flask, or any standard python dependencies can be managed at requirements.txt level.

Reply:
fixed